test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift

```diff
@@ -120,23 +120,33 @@ class ViewController: UIViewController, WKUIDelegate, UIDocumentPickerDelegate {
     private var webView : WKWebView? = nil
     private var configData: JSON = [:]
     private var authClient: AuthorizationClient? = nil
-
+    private var documentCompletion : ((URL?) -> Void)? = nil
```

Comment:
I believe that `()` is preferred over `Void` in Swift. I don't know why that is, but something that was activated in SYNCHRO would spit out a warning any time `Void` was used instead of `()`.

Reply:
Never mind.